sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -74,7 +85,27 @@ internal class RadarApiClient(
         )
     }
 
-    internal fun getConfig() {
+    private fun parseMeta(res: JSONObject?): RadarTrackingOptions? {
+        var meta: JSONObject? = null
+        if (res?.has("meta") == true) {
+            meta = res.getJSONObject("meta")
+        }
+
+        if (meta?.has("config") == true) {
+            val config = meta.getJSONObject("config")
+            RadarSettings.setConfig(context, config)
```

Comment:
do we need to do any validation of the `config` object on the client before setting it?

Reply:
I kept the existing behaviour from before. probably, but it's also not used